cornflow-server/cornflow/endpoints/login.py

```diff
@@ -141,57 +164,38 @@ def auth_oid_authenticate(
         self, token: str = None, username: str = None, password: str = None
     ):
         """
-        Method  in charge of performing the log in with the token issued by an Open ID provider.
-        It has an exception and thus accepts username and password for service users if needed.
+        Method in charge of performing the authentication using OpenID Connect tokens.
+        Supports any OIDC provider configured via provider_url.
 
-        :param str token: the token that the user has obtained from the Open ID provider
-        :param str username: the username of the user to log in
-        :param str password: the password of the user to log in
-        :return: the user object or it raises an error if it has not been possible to log in
+        :param str token: the JWT token from the OIDC provider
+        :param str username: username for service users
+        :param str password: password for service users
+        :return: the user object, or it raises an error if it has not been possible to log in
         :rtype: :class:`UserModel`
         """
-
+        print("[auth_oid_authenticate] Starting OpenID authentication")
         if token:
```

Comment:
There are several prints in this endpoint, they should either be removed or changed to logging